### ISSUE TITLE
A3C hyper-parameters

### ISSUE THREAD

**Issue by sxjzwq** [ppwwyyxx/tensorpack#326]
Hi, I am using you A3C code to train a model for an atari game, WizardOfWor-v0.
I have two GPUs (K40c) and more than 20 CPU cores and my setting is 

BATCH_SIZE = 128
PREDICT_BATCH_SIZE = 30     # batch for efficient forward
SIMULATOR_PROC = 120
PREDICTOR_THREAD_PER_GPU = 6

I am not sure whether this is a proper setting and my question is will these parameters affect the results a lot? Do you have any suggestion (experience) of setting these parameters so that I can reproduce your results.

I also tried to train the model by using your Model Zoo provided model as the initialisation with the above setting (I did decrease the lr-rate). After nearly 50 epochs, the results became worse. Your original model is 3824 but after my fine-tuning it became around 2000. So I am thinking whether it is caused by my above hyper-parameter setting?

Thanks!

**Comment by ppwwyyxx** [comment]
These parameters should only affect speed. I don't know why would you fine-tune to a worse score, but in my experience vanilla a3c models are not very stable anyway.

**Comment by sxjzwq** [comment]
Thanks.

I find it can be diverge easily on some difficult games (such as WizardOfWor). Did you have similar situation? How did you deal with that? continue training until it converge at some point or restart the training process?

You mentioned that you train 2 days to get those models in your Model Zoo, how many epochs of those model?

**Comment by ppwwyyxx** [comment]
Some parameters (learning rate, entropy factor) may need some tuning for difficult games. These models were trained a year ago and I don't remember much details. 2 days might be somewhere around 2million steps?
